2_observations/src/munge_surface_area_data.R

```diff
@@ -0,0 +1,65 @@
+read_monthly_surface_area_data <- function(res_sf_dir) {
+
+  # check if data exists
+  # if not, throw error indicating how to download
+  if (!file.exists("2_observations/in/009130")) {
+    stop("You must download reservoir monthly shapefile folder (0) data from:
+         \nhttp://umnlcc.cs.umn.edu/realsat
+         \nunzip the folder,
+         \nthen locate folder '009130' and unzip the folder into '2_observations/in'.")
+  }
+
+  if (!file.exists("2_observations/in/573567")) {
+    stop("You must download reservoir monthly shapefile folder (7) data from:
+         \nhttp://umnlcc.cs.umn.edu/realsat
+         \nunzip the folder,
+         \nthen locate folder '573567' and unzip the folder into '2_observations/in'.")
```

Comment:
This call prints a double-spaced error message:
```
Error: You must download reservoir monthly shapefile folder (7) data from:
         
http://umnlcc.cs.umn.edu/realsat
         
unzip the folder,
         
then locate folder '573567' and unzip the folder into '2_observations/in'.
```
Can you make it single-spaced?

If you want to use several lines for the message, which seems fine/good to me, consider something like `stop(paste('line 1', 'line 2', 'line 3', sep='\n    '))`

Reply:
As a further challenge, can you use DRYer code in lines 5-17 such that you iterate over `c('009130','573567')` and c(0,7) and then only have to write out `if` and `stop` once each? Ask for help (from Hayley) if necessary.

Reply:
It may be useful to note that the corresponding shapefile folder is numbered according to the last digit of the ID you're looking for - Rasha probably knows this but Hayley may not.